src/libsync/accessmanager.cpp

```diff
@@ -62,6 +62,16 @@ QNetworkReply *AccessManager::createRequest(QNetworkAccessManager::Operation op,
     // Some firewalls reject requests that have a "User-Agent" but no "Accept" header
     newRequest.setRawHeader(QByteArrayLiteral("Accept"), QByteArrayLiteral("*/*"));
 
+    // Set the language, so messages from the server are localised correctly.
+    QLocale locale;
+    QByteArray acceptedLanguage;
+    if (locale.language() == QLocale::AnyLanguage) {
```

Comment:
Can the clients locale really be "AnyLanguage"?

Reply:
I assume since this is the `0` value of the enum, it's the fallback. However, I don't see a point in handling this case either, since I'd consider *not* having a valid locale set globally an error. This would only make sense if the language was a property on the request.

Reply:
ok, removed it.